omniscidb/Tests/ResultSetBaselineRadixSortTest.cpp

```diff
@@ -39,7 +39,10 @@ extern std::unique_ptr<CudaMgr_Namespace::CudaMgr> g_cuda_mgr;
 #include <numeric>
 #include <random>
 
-extern bool g_is_test_env;
+#ifdef _WIN32
```

Comment:
Do we have to add this to all extern declarations? Maybe make EXTERN macro then which would work for both Linux and Windows?